### PR TITLE
LATCH-1676 Feedback: Allowing 3 buttons at the same time

### PR DESCRIPTION
**Pull request by salavert**
## 🎟️ **Jira ticket**
[LATCH-1676](https://jira.tid.es/browse/LATCH-1676) Feedback allow primary, secondary and link buttons at the same time

## 🥅 **What's the goal?**
As required by design we need to allow 3 buttons at the same time: primary, secondary and link

Storybook: https://mistica-web.vercel.app/?path=/story/patterns-feedback-feedbackscreen--feedback-screen-story

## 🧪 **How can I verify this?**

Check new [readme](https://github.com/Telefonica/mistica-ios/tree/LATCH-1676-feedback-buttons/Sources/MisticaSwiftUI/Components/Feedback#feedbacks)

### CODE REVIEW

File: Sources/MisticaSwiftUI/Components/Callout/Callout.swift
Comment (salavert) [inline Sources/MisticaSwiftUI/Components/Callout/Callout.swift:1]: Small bug I found

**Review by yceballost — APPROVED** [review]

**Review by L-Trujillo26 — APPROVED** [review]

File: Sources/MisticaSwiftUI/Components/Feedback/docs/images/feedback.png
Comment (DavidMarinCalleja) [inline Sources/MisticaSwiftUI/Components/Feedback/docs/images/feedback.png:1]: the icon is different, isn't it? is expected? @salavert

Comment (salavert) [reply Sources/MisticaSwiftUI/Components/Feedback/docs/images/feedback.png:1]: informative/error/success have fixed images but feedback allows any image

**Review by DavidMarinCalleja — APPROVED** [review]

**Pull request merged**